libs/api/domains/auth-admin/src/lib/tenant/tenant.resolver.ts

```diff
@@ -39,4 +39,9 @@ export class TenantResolver {
   resolveAvailableEnvironments(@Parent() tenant: Tenant): Environment[] {
     return tenant.environments.map((env) => env.environment)
   }
+
+  @Query(() => Tenant, { name: 'authAdminTenantById' })
```

Comment:
I think our gql naming convention is to have this as singular `authAdminTenant`.
```suggestion
  @Query(() => Tenant, { name: 'authAdminTenant' })
```